app/src/main/java/com/telnyx/webrtc/sdk/ui/MainActivity.kt

```diff
@@ -215,6 +217,13 @@ class MainActivity : AppCompatActivity() {
                             Toast.LENGTH_SHORT
                         ).show()
                         progress_indicator_id.visibility = View.INVISIBLE
+                        progress_indicator_id.visibility = View.INVISIBLE
```

Comment:
is this line duplicated with the one above?

Reply:
oh yes, I'll remove it